src/components/Accordion.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import './accordion.css';
+
+class Accordion extends Component {
+    constructor(props){
+        super(props);
+        this.state = {
+            open: false
+        }
+
+        this.togglePanel = this.togglePanel.bind(this);
+        }
+        
+        togglePanel(e){
+        this.setState(prevState => ({ open: !prevState.open }))
+        }
+        
+    render() {
+        const title = this.props.title;
+        const children = this.props.children;
```

Comment:
il poti face cu destructuring
`const { title } = this.props`

si il poti pune intr-o singura linie cele care sunt din this.props, si intr-o alta linie cele care sunt din this.state.

`const { firstName, lastName } = this.props`

Again, daca ai avea setat eslint, aici ar face complains